TestTools/StreamChatTestTools/Mocks/StreamChat/Workers/ChannelUpdater_Mock.swift

```diff
@@ -26,6 +26,7 @@ final class ChannelUpdater_Mock: ChannelUpdater {
 
     @Atomic var deleteChannel_cid: ChannelId?
     @Atomic var deleteChannel_completion: ((Error?) -> Void)?
+    @Atomic var deleteChannel_completion_next_result: Result<Void, Error>?
```

Comment:
These are `Atomic` but the mock actually never uses the `mutate` function so all in all, we never use the internal locking what `Atomic` provides. Probably makes sense to delete all of these `Atomic` property wrappers.